modules/TestUtils.js

```diff
@@ -88,7 +88,7 @@ export const objectContains = (object, value, compareValues) => {
     compareValues = isEqual
 
   return Object.keys(value).every(k => {
-    if (isObject(object[k])) {
+    if (isObject(object[k]) && Object.keys(value[k]).length > 0) {
```

Comment:
Thanks for replying. I am not sure what do you mean. Do you mean to check `Object.keys(value).length > 0` before entering `Object.keys(value).every()`?